### PR TITLE
Kb/js code string concat

### PR DESCRIPTION
**Pull request by kurt-r2c**
@LewisArdern

### CODE REVIEW

File: javascript/lang/security/audit/code-string-concat.yaml [hunk 1]
```diff
@@ -1,34 +1,76 @@
...
     metadata:
-      owasp: "A1: Injection"
+      confidence: HIGH
+      owasp: 
+        - "A01:2017 - Injection"
+        - "A03:2021 - Injection"
       cwe: "CWE-95: Improper Neutralization of Directives in Dynamically Evaluated Code ('Eval Injection')"
       category: security
       technology:
         - node.js
```
Comment (LewisArdern) [inline javascript/lang/security/audit/code-string-concat.yaml:22]: Express
Nextjs

File: javascript/lang/security/audit/code-string-concat.yaml [hunk 2]
```diff
@@ -1,34 +1,76 @@
...
-              regex: .*(function|Function|eval|\([^\(\)]*\)\s*=>).*
-      - patterns:
-          - pattern-inside: |
-              `...${...}...`
-          - pattern-regex: function|Function|eval|\([^\(\)]*\)\s*=>
+        - pattern: |
+            $M(...)
+        - metavariable-regex:
+            metavariable: $M
+            regex: (eval|exec)
```
Comment (LewisArdern) [inline javascript/lang/security/audit/code-string-concat.yaml]: child_process.exec? https://nodejs.org/api/child_process.html probably just keep to eval and save the others for a new rule.. idk

Comment (kurt-r2c) [reply javascript/lang/security/audit/code-string-concat.yaml]: removed. can always add a new rule for child_process later

File: javascript/lang/security/audit/code-string-concat.yaml (hunk 1 above)
Comment (kurt-r2c) [reply javascript/lang/security/audit/code-string-concat.yaml:22]: updated

**Review by LewisArdern — APPROVED** [review]
lgtm

**Pull request merged**